workout-app/src/components/weekpot.tsx

```diff
@@ -1,11 +1,71 @@
-import React from 'react'
+import React, { useState } from 'react'
+import "@fontsource/reenie-beanie"
+
+const weekpot = ({toggleBoard, handleToggle}) => {
+
+  const nameMapping = {
+    1: 'Sandeep Suri',
```

Comment:
Great Work Sunny. I'm assuming once we generate the intake form you will introduce logic to make this dynamic right?